hpobench/__version__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.5dev'
```

Comment:
This should be only "0.0.5"

Reply:
Thought about this too. But if I change it here, then the dev branch loses its _dev tag. So I thought, that we have to replace it directly on the master later, or is there another simple way I don't see? :-)

Reply:
No, we do it here and then directly push a new commit to dev initializing the new version (see here: https://github.com/automl/HPOBench/commit/6ffe1a44d233d57aadee2624acc60cc27dc1c3ec)